src/unlimited.cpp

```diff
@@ -415,9 +434,17 @@ std::string FormatCoinbaseMessage(const std::vector<std::string>& comments,const
 CNode* FindLikelyNode(const std::string& addrName)
```

Comment:
There is already a FindNode function.  The name of this function is based off of that one.  This function allows partial matches.  Currently the function is documented in the header file, although documenting the wildcard addition is needed.